certify-service/src/main/java/io/mosip/certify/services/LoggerAuditService.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package io.mosip.certify.services;
```

Comment:
can we keep this little isolated ? like io.mosip.certify.plugin.impl ?

Reply:
moved this to plugin impl folder